docs/HOWTO_add_routing_tiles.MD

```diff
@@ -0,0 +1,62 @@
+HOWTO_Add_routing tiles for Wahoo Elemnt ROAM
+
+Wahoo Elemnt ROAM is using OSM based map, and the base map tiles can be either downloaded from Wahoo update server (url: https://cdn.wahooligan.com/wahoo-maps/{version}/tiles/{zoom}/{x}/{y}.map.lzma; latest map version = 12, zoom level =8, x, y = map tile index/coordinates) or generated by using WahooMapsCreator (https://github.com/treee111/wahooMapsCreator). Besides the base map tiles, routing tiles are also required to support routing/navigation features (i.e. turn by turn support for Strava route). And it seems Valhalla (https://valhalla.readthedocs.io/en/latest/api/turn-by-turn/overview/) is used by Wahoo Elemnt ROAM as routing services. Following are the detailed instructions for getting routing tiles.
+
+1. Directory hierarchy in Wahoo Elelmnt ROAM device
+maps
+   | - routing
+   |      | - routing-config.json
+   |      | - 2
+   |           |- 000
+   |               | - xxx
+   |               |    |- yyy.gph.lzma
+   |               |    | -  ……
+   |               |    | - yyy.gph.lzma
+   |               | - ……
+   |               | - ……
+   |               | - ……
+   |               | - xxx
+   |               |    | - yyy.gph.lzma
+   |               |    | -  ……
+   |               |    | - yyy.gph.lzma
+   | - tiles
+	| - 8
+	    | - xxx
+	    |    | - yyy.map.lzma
+	    |    | -  ……
+	    |    | - yyy.map.lzma
+	    | - ……
+	    | - ……
+	    | - ……
+	    | - xxx
+	    |    | - yyy.map.lzma
+	    |    | -  ……
+	    |    | - yyy.map.lzma
+
+
+2. How to get routing tiles?
+	a. Option a: download from wahoo update server
+	First of all, index numbers for the routing tiles of target country are required before trying to download them from wahoo update server. Since wahoo use the routing tiles ID ("routing_coordinates") instead of index in mappack-gzip.json, the python script from valhalla are required to decode the ID  into six digital index (first three are the folder name and last three are filename).
+
+	Following are the python code for the decode function. More details and full python script can be found from: https://valhalla.readthedocs.io/en/latest/tiles/
+
+	#!/usr/bin/env python
+
+	LEVEL_BITS = 3 
+	TILE_INDEX_BITS = 22 
+	TILE_INDEX_MASK = (2**TILE_INDEX_BITS) - 1
+	def get_tile_index(id): 
+	  return (id >> LEVEL_BITS) & TILE_INDEX_MASK
+
+	After decoding the routing tile IDs into index, it is fairly easy to download all routing tiles from wahoo update server. Following are the URL:
+		https://cdn.wahooligan.com/wahoo-maps/{version}/routing/tiles/{level}/000/{x}/{y}.gph.lzma
+	And the latest map version is 12; level is 2; x, y are both 3 digitals map tile index.
+
+	Note:
+			i. Please use 3 digitals for x and y, and add "0" if x or y is less than 100. e.g. use "003" instead "3" when trying to download them
+			ii. Some tiles may be missing, and the reason and impact are unclear. But it still can work.

```

Comment:
the indentation is not reflected in Github.
Lists or empty lines could help in my opinion